addons/project/views/account_analytic_account_line_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_view_account_analytic_line_graph" model="ir.ui.view">
+        <field name="name">account.analytic.line.graph</field>
+        <field name="model">account.analytic.line</field>
```

Comment:
Sorry, maybe I did a mistake in my previous review, but the file name should be `{model_name}_views.xml` and so `account_analytic_line_views.xml`. 😇